### PR TITLE
[MRG+1] [DOC] update step function documentation

### PR DESCRIPTION
**Pull request by hkucukdereli**
Hello from the ASPP School. These are our "improvements" to the documentation. @NelleV

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -1823,7 +1828,10 @@ def step(self, x, y, *args, **kwargs):
           If 'mid', the jumps in *y* occur half-way between the
           *x*-values.
 
-        Return value is a list of lines that were added.
+        Note
+        ----
+        Additional parameters are the same as those for :func:`~matplotlib.pyplot.plot`.
```
Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: This line is slightly too long for our code standard (max 79 character). Can you break it after "for"?

**Comment by NelleV** [comment]
Apart from my comment (and the missing test), I think it is good. If you update the pull request, we'll see if the tests pass!

Thanks for the patch.
N

**Comment by QuLogic** [comment]
It appears you have not [set the config for your git client](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#Your-Identity). All your commits are attributed to "Your Name <you@example.com>", which is probably not what you want.

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -1802,28 +1802,37 @@ def step(self, x, y, *args, **kwargs):
...
+        out : list
+            List of lines that were added.
 
-        *where*: [ 'pre' | 'post' | 'mid'  ]
-          If 'pre' (the default), the interval from x[i] to x[i+1] has level
-          y[i+1].
+        Other parameters
+        ---------------------
+        where : [ 'pre' | 'post' | 'mid'  ]
+          If 'pre' (the default), the interval from
```
Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: There is a trailing space at the end of this line. Can you please remove it?

File: lib/matplotlib/axes/_axes.py [hunk 3]
```diff
@@ -1802,28 +1802,37 @@ def step(self, x, y, *args, **kwargs):
...
 
           If 'post', that interval has level y[i].
 
           If 'mid', the jumps in *y* occur half-way between the
           *x*-values.
 
-        Return value is a list of lines that were added.
+        Note
+        ----
+        Additional parameters are the same as those for
```
Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: There is a trailing space at the end of this line. Can you please remove it?

**Comment by hkucukdereli** [comment]
Here's another try. I believe I cleaned the trailing space and tabs. @NelleV

**Comment by NelleV** [comment]
Wohoo ! The tests pass! CongratsT

**Comment by hkucukdereli** [comment]
Yay!

**Comment by NelleV** [comment]
Here is how it looks (the top only):

step

**Comment by QuLogic** [comment]
The first two commits are still attributed to `Your Name <you@example.com>` and the last commit has a typo in your email address (.comm instead of .com).

If you are not comfortable with rebasing to correct that issue, we can take care of it for you before merging if you let us know the name and email to use.

**Comment by hkucukdereli** [comment]
@QuLogic It'd be great if you guys can rebase it for me. Thanks a lot. Below are my details:
Hakan Kucukdereli
hkucukdereli@gmail.com

**Comment by NelleV** [comment]
I've rebased and created a new PR. I'll merge as soon as the tests finish.
Thanks @hkucukdereli for your contribution?

**Pull request closed without merging**